.travis.yml

```diff
@@ -17,18 +18,25 @@ before_script:
 # Compute the rust version we use. We do not use "language: rust" to have more control here.
 - |
   if [[ "$TRAVIS_EVENT_TYPE" == cron ]]; then
-    RUST_TOOLCHAIN=nightly
+    RUSTC_HASH=$(git ls-remote https://github.com/rust-lang/rust.git master | awk '{print $1}')
   else
-    RUST_TOOLCHAIN=$(cat rust-version)
+    RUSTC_HASH=$(cat rust-version)
   fi
 # install Rust
-- curl https://build.travis-ci.org/files/rustup-init.sh -sSf | sh -s -- -y --default-toolchain "$RUST_TOOLCHAIN"
+- curl https://build.travis-ci.org/files/rustup-init.sh -sSf | sh -s -- -y --default-toolchain stable
```

Comment:
looks like we need `nightly` here

Reply:
I used `stable` in purpose to get better caching.

But we also need to install the `cargo` component with `rustup-toolchain-install-master`.